pennylane_lightning/core/src/gates/Gates.hpp

```diff
@@ -1238,6 +1257,8 @@ std::vector<ComplexT<T>> getMatrix(const GateOperation gate_op,
         return getCZ<ComplexT, T>();
     case GateOperation::SWAP:
         return getSWAP<ComplexT, T>();
+    case GateOperation::ISWAP:
+        return getISWAP<ComplexT, T>();
```

Comment:
Is this correct regardless of the value of `inverse`?

Reply:
No, take inverse parameter into consideration, the adjoint matrix is different. It may take a parameter (inverse?-1:1) and multiply it to the image elements in the matrix.